### PR TITLE
Supporting Array[File] inputs for the JES backend.

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: src/main/scala/cromwell/engine/backend/jes/JesBackend.scala [hunk 1]
```diff
@@ -199,11 +199,24 @@ class JesBackend extends Backend with LazyLogging {
       case (fileTag: String, location: WdlFile) =>
         logger.info(s"${makeTag(backendCall)}: $fileTag -> ${location.valueString}")
         Seq(mkInput(fileTag, location))
-      case (fileTag: String, location: WdlArray) => location.value.collect { case f: WdlFile => mkInput(fileTag, f) }
+      case (fileTag: String, location: WdlArray) => inputsFromArray(fileTag, location.value, backendCall)
```
Comment (scottfrazer) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: Why `fileTag` and not `fqn`?

Comment (scottfrazer) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: Also maybe `array: WdlArray` instead of `location: WdlArray`

**Comment by mcovarr** [comment]
:skull: :boom: :bomb: :zap:  This PR is targeting master!  :skull: :boom: :bomb: :zap:

File: src/main/scala/cromwell/engine/backend/jes/JesBackend.scala (hunk 1 above)
Comment (scottfrazer) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: as discussed in person, fileTag really is a locallyQualifiedName

**Comment by cjllanwarne** [comment]
Let us never speak of this again.

**Comment by mcovarr** [comment]
:speak_no_evil:

**Pull request closed without merging**